nixos/modules/services/monitoring/prometheus/exporters/rtl_433.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, options }:
+
+with lib;
```

Comment:
I'm in general not a big fan of "with", especially for something as broad as "lib". In some cases like for `meta = with lib; {` it's fairly well accepted but here it's not actually saving much imo.

Reply:
Sure, removed.

Reply:
for future reference (but not relevant for this pr):
instead of using `with lib;` to pull everything from `lib` in scope or specifying `lib.` everywhere, one can also pull in only the needed defintions as follows:
```nix
{ lib, ... }:
let
  inherit (lib)
    mkOption
    # ... more stuff from lib
    types;
in
{
  options.example-module-option = mkOption {
    type = types.bool;
    # ...
  };
  # ...
}
```